### ISSUE TITLE
Make arrows + close customizable

### ISSUE THREAD

**Issue by webgefrickel** [feimosi/baguetteBox.js#108]
Just as a small idea: would it be possible to make the markup for the arrows and the close-function customizable? So basically just move those three vars into the options-object? This way I could more easily modify the look and feel of baguetteBox.

I am happy to create a pull request if you think this is a good idea.

**Comment by feimosi** [comment]
Hi, sure, make a PR. I can then take a look and assess whether it fits well.

**Comment by xavierve** [comment]
yes, having control over action buttons will be great. Thanks for great code!

**Comment by jackbarham** [comment]
This would be awesome. SlickSlider has an option, such as:

`rightArrow: <i class="fa fa-arrow-right"></i>`

Which is really useful :)

**Comment by Olli** [comment]
https://github.com/feimosi/baguetteBox.js/pull/204
